src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -1027,6 +1027,110 @@ public HyperwalletPayPalAccount getPayPalAccount(String userToken, String payPal
         return listPayPalAccounts(userToken, null);
     }
 
+    /**
+     * Deactivate PayPal Account
+     *
+     * @param userToken          User token
+     * @param payPalAccountToken PayPal Account token
+     * @return HyperwalletStatusTransition deactivated PayPal account
+     */
+    public HyperwalletStatusTransition deactivatePayPalAccount(String userToken, String payPalAccountToken) {
+        return deactivatePayPalAccount(userToken, payPalAccountToken, null);
+    }
+
+    /**
+     * Deactivate a PayPal Account
+     *
+     * @param userToken          User token
+     * @param payPalAccountToken PayPal Account token
+     * @return The status transition
+     */
+    public HyperwalletStatusTransition deactivatePayPalAccount(String userToken, String payPalAccountToken, String notes) {
+        return createPayPalAccountStatusTransition(userToken,
+                payPalAccountToken,
+                new HyperwalletStatusTransition(HyperwalletStatusTransition.Status.DE_ACTIVATED).notes(notes));
+    }
+
+    /**
+     * Create PayPal Account Status Transition
+     *
+     * @param userToken          User token
+     * @param payPalAccountToken PayPal Account token
+     * @param transition         Status transition information
+     * @return HyperwalletStatusTransition new status for PayPal Account
+     */
+    public HyperwalletStatusTransition createPayPalAccountStatusTransition(String userToken, String payPalAccountToken, HyperwalletStatusTransition transition) {
+        if (transition == null) {
+            throw new HyperwalletException("Transition is required");
+        }
+        if (StringUtils.isEmpty(userToken)) {
+            throw new HyperwalletException("User token is required");
+        }
+        if (StringUtils.isEmpty(payPalAccountToken)) {
+            throw new HyperwalletException("PayPal Account token is required");
+        }
+        if (!StringUtils.isEmpty(transition.getToken())) {
+            throw new HyperwalletException("Status Transition token may not be present");
+        }
+        transition = copy(transition);
+        transition.setCreatedOn(null);
+        transition.setFromStatus(null);
+        transition.setToStatus(null);
+        return apiClient.post(url + "/users/" + userToken + "/PayPal-accounts/" + payPalAccountToken + "/status-transitions", transition, HyperwalletStatusTransition.class);
```

Comment:
why do we need upper case in url, instead of "/paypal-accounts/"

Reply:
I was confused with incorrect example, I have already corrected it